app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  after_save :update_likes_counter
+  belongs_to :user
```

Comment:
- [ ] it would be better if you change `user` to `author`. as in the diagram in order to match the requirements you can do that like the below example: 
```ruby
`belongs_to :author, class_name: 'User'`
```

Reply:
Changes required have been made ✅